src/chatlog/chatmessage.cpp

```diff
@@ -234,5 +320,5 @@ QString ChatMessage::detectQuotes(const QString& str, MessageType type)
 
 QString ChatMessage::wrapDiv(const QString &str, const QString &div)
 {
-    return QString("<div class=%1>%2</div>").arg(div, str);
+    return QString("<p class=%1>%2</p>").arg(div, /*QChar(0x200E) + */str);
```

Comment:
This should fix arabic font display and was commented because of https://github.com/tux3/qTox/pull/2013#issuecomment-122404113 which isn't fixed.